commons-vfs2-jackrabbit2/src/main/java/org/apache/commons/vfs2/provider/webdav4/Webdav4FileObject.java

```diff
@@ -487,11 +488,17 @@ private HttpResponse executeRequest(final HttpUriRequest request) throws FileSys
             response = executeHttpUriRequest(request);
             final int status = response.getStatusLine().getStatusCode();
             if (status == HttpURLConnection.HTTP_NOT_FOUND || status == HttpURLConnection.HTTP_GONE) {
+                EntityUtils.consume(response.getEntity());
                 throw new FileNotFoundException(request.getURI());
             }
 
             if (request instanceof BaseDavRequest) {
-                ((BaseDavRequest) request).checkSuccess(response);
+                try {
+                    ((BaseDavRequest) request).checkSuccess(response);
+                } catch(DavException e){
+                    EntityUtils.consume(response.getEntity());

```

Comment:
Hello @beise 
Isn't the response consumed in `org.apache.jackrabbit.webdav.client.methods.BaseDavRequest.getResponseException(HttpResponse)` by calling `org.apache.jackrabbit.webdav.client.methods.BaseDavRequest.getResponseBodyAsDocument(HttpEntity)` before  throwing `DavException`?

Reply:
Hi @garydgregory 
you are right! This change is indeed unnecessary.